src/components/avatar/avatar.html

```diff
@@ -0,0 +1,59 @@
+<article>
```

Comment:
all `components/x/x.html` files are extracted from existing basic/components/containers files. You don't have to focus on reading these